wbooks/src/app/components/AppNavigator/index.tsx

```diff
@@ -1,32 +1,39 @@
 import React from 'react';
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import { createStackNavigator } from '@react-navigation/stack';
+import SearchIcon from '@components/SearchIcon';
 import TabIcon from '@components/TabIcon';
 import colors from '@constants/colors';
 import SCREENS, { SCREEN_TITLES } from '@constants/screens';
 import { BottomTabParamList, LibraryStackParamList } from '@interfaces/navigatorParamLists';
 import BookDetails from '@screens/BookDetails';
 import Library from '@screens/Library';
+import SearchScreen from '@screens/SearchScreen';
 import WishList from '@screens/WishList';
 
-import customHeader from './config';
+import mainHeader, { searchHeader } from './config';
 
 const TabNavigator = createBottomTabNavigator<BottomTabParamList>();
 const LibraryStackNavigator = createStackNavigator<LibraryStackParamList>();
 
 function LibraryStackScreen() {
   return (
-    <LibraryStackNavigator.Navigator screenOptions={customHeader as object}>
+    <LibraryStackNavigator.Navigator screenOptions={mainHeader as object}>
```

Comment:
Revert changes!

Reply:
I'm no longer using and spreading the `sharedHeaderProperties`, so now the `mainHeader` has all the main properties, and the other headers only contain the customized features.